pkg/fakerp/fakerp.go

```diff
@@ -54,6 +54,13 @@ func CreateOrUpdate(ctx context.Context, oc *v20180930preview.OpenShiftManagedCl
 
 		log.Info("merge old and new config")
 		p.MergeConfig(ctx, cs, oldCs)
+	} else {
```

Comment:
We should probably create the DNS records only after the creation is complete.

Reply:
I got the impression that RP creates it before. Need to check

Reply:
If that's the case, then fine.